librz/arch/p_gnu/asm/asm_vax_gnu.c

```diff
@@ -21,62 +21,64 @@ static const ut8 *bytes = NULL;
 static int bytes_size = 0;
 
 static int vax_buffer_read_memory(bfd_vma memaddr, bfd_byte *myaddr, ut32 length, struct disassemble_info *info) {
-	int delta = (memaddr - Offset);
-	if (delta < 0) {
-		return -1; // disable backward reads
-	}
-	if (delta > length) {
-		return -1;
-	}
-	memcpy(myaddr, bytes + delta, RZ_MIN(length, bytes_size));
-	return 0;
+    int delta = (memaddr - Offset);
```

Comment:
You need t run `clang-format` on those. The easiest way is `git-clang-format` once all the files are staged. Rizin uses currently `clang-format-16`